.github/workflows/ci.yml

```diff
@@ -57,3 +59,27 @@ jobs:
         env:
           CI_NAME: github-actions
           COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+      - name: Pack
+        run: dotnet pack src -c Release --no-build --output nuget
+      - name: 'Upload Artifact'
+        uses: actions/upload-artifact@v2
```

Comment:
WDYT about only conditionally uploading the artifact if we know we're publishing to avoid unnecessary storage?

Reply:
Artifacts are cleaned after 90 days by github. I've never seen artifact storage becoming a problem.